SPECS/golang/golang-1.15.spec

```diff
@@ -22,7 +22,6 @@ URL:            https://golang.org
 Source0:        https://dl.google.com/go/go%{version}.src.tar.gz
 Source1:        https://dl.google.com/go/go1.4-bootstrap-20171003.tar.gz
 Patch0:         go14_bootstrap_aarch64.patch
```

Comment:
Could you also now rename the patch, so it doesn't mention "aarch64"?

Reply:
The patch allows compiling arm32 go 1.4 on aarch64, so the architecture should still be in the title, IMO

Reply:
I see. I misunderstood the discussion about where this patch is needed and though that it's required for all architectures, but now I see that what was meant, was that it simply does no harm when applied for all builds.